pr-data.csv

```diff
@@ -6252,6 +6252,7 @@ https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,qu
 https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,querydsl-hibernate-search,com.querydsl.hibernate.search.SearchQueryTest.ordering,NIO;NOD,Accepted,https://github.com/querydsl/querydsl/pull/2658,
 https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,querydsl-hibernate-search,com.querydsl.hibernate.search.SearchQueryTest.paging,NOD,Accepted,https://github.com/querydsl/querydsl/pull/2658,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/45
 https://github.com/querydsl/querydsl,2bf234caf78549813a1e0f44d9c30ecc5ef734e3,querydsl-hibernate-search,com.querydsl.hibernate.search.SearchQueryTest.singleResult,OD-Brit,,,
+https://github.com/questdb/questdb,802e00fbbab89b05057fe1850e4579aad106448c,core/src/test/java/io/questdb/test/griffin/engine/functions/regex,io.questdb.test.griffin.engine.functions.regex.RegexpReplaceStrFunctionFactoryTest.testStackOverFlowError,ID,Opened,https://github.com/questdb/questdb/pull/6242,https://github.com/questdb/questdb/issues/4139
```

Comment:
Why do you think that this test is `ID`? Does it fail with NonDex shuffling but passes without shuffling?